lib/perl/Genome/Model/Tools/DetectVariants2/Result/DetectionBase.pm

```diff
@@ -405,6 +406,7 @@ sub vcf_result_params {
         aligned_reads_sample => $aligned_reads_sample,
         ($control_aligned_reads_sample? (control_aligned_reads_sample => $control_aligned_reads_sample) : ()),
 
+        #users => $users, #Needed if get() inside get_vcf_result becomes get_with_lock()
```

Comment:
If someone tries to change it to `get_with_lock()` tests will break with an error from `Params::Validate`.  I could leave this comment out--it is a bit silly to add commented-out code.